src/menu_events.cpp

```diff
@@ -1411,15 +1411,25 @@ void console_handler::do_refresh()
 
 void console_handler::do_droid()
 {
-	// :droid [<side> [on/off/full]]
-	const std::string side_s = get_arg(1);
+	// :droid [side] [on/off/full]
```

Comment:
`<side>` means that you have to replace this with an appropriate value. In contrast to `on/off/full` (which should really be `on|off|full]` as in the help) where those are the literal values to use.

If you must change how the command works then please do it in a separate commit.